staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -170,13 +170,10 @@ func (p *restrictedProfile) Apply(pod *corev1.Pod, containerName string, target
 	dropCapabilities(pod, containerName)
 
 	switch style {
-	case node:
-		clearSecurityContext(pod, containerName)
-
 	case podCopy:
 		shareProcessNamespace(pod)
 
-	case ephemeral:
+	case ephemeral, node:
```

Comment:
Why we are not clearing securityContext anymore for `node`?

Reply:
To ensure that the effects of having set the running as non-root & drop all capabilities are not nullified.